docs/performance.md

```diff
@@ -368,3 +369,30 @@ MiB Swap:      0.0 total,      0.0 free,      0.0 used.  30354.7 avail Mem
  272507 al        20   0   24.3g   1.4g 336236 S  97.0   4.3   4:42.73 posix
 
 ```
+
+### S3 + MySQL
+
+S3 + MySQL performance number will highly depend on the S3 and MySQL setup. If
+you have the opportunity to run load tests with different relevant setups, we'd
+love to hear about them: [get in touch](../README.md#wave-contact)!
+
+#### MinIO MariaDB (from IPng Networks)
+
+This test was performed by [IPng Networks](https://ipng.ch/), more details are
+available on their [blog post](https://ipng.ch/s/articles/2025/07/26/certificate-transparency-part-1/).
+Kudos to IPng and Pim!
+
+This test was performed with on a Dell R630s running with two Xeon E5-2640 v4
+CPUs, with 20 cores, 40 threads, and 512GB of DDR4 memory. The machine had a
+SAS controller with 6pcs of 1.92TB enterprise storage (Samsung part
+number P1633N19).
+
+The log, hammer, MinIO and MariaDB were running on the same machine.
+
+TesseraCT sustained 500 write qps for a few hours, with:
+
+* TesseraCT using about 2.9 CPUs/s
+* MariaDB using 0.3 CPUs/s
+* The hammer using 6.0 CPUs/s
```

Comment:
We can't change the published numbers, but this number is already likely improved since #456.

Reply:
Good point, I added some indication about when these test were done.